src/SSVOpenHexagon/Core/HGGraphics.cpp

```diff
@@ -330,8 +340,16 @@ void HexagonGame::updateText()
 
         os.str("");
 
-        os << formatTime(rf._played_frametime / 60.0) << "s BY "
-           << rf._player_name << '\n';
+        if (levelStatus.scoreOverridden)
+        {
+            os << formatTime(rf._played_score) << " BY "
+               << rf._player_name << '\n';
+        }
+        else
+        {
+            os << formatTime(rf._played_score / 60.0) << "s BY "
+               << rf._player_name << '\n';
+        }
```

Comment:
You can extract the "BY ..." portion of the `os <<` streaming out of the if statement, as it's the same in both branches.

Reply:
Yeah, I can see how this implementation can be much cleaner. I can go ahead and fix this real quick.